zig build gen fails on zig 0.11.0-dev.2704+83970b6d9

### Zig Version

v0.11.0-dev.2704+83970b6d9

### Zig Language Server Version

3ae56929fcc695fb6fbd2c74f86feb7d4890e368

### Steps to Reproduce

1. build with zig 0.11.0-dev.2666+1e207f1ed
2. run `zig build gen -- --generate-version-data master`
3. rebuild with zig 0.11.0-dev.2704+83970b6d9
4. run `zig build gen -- --generate-version-data master`

### Expected Behavior

The build succeeded

### Actual Behavior

gen transitive failure
└─ run zls_gen transitive failure
   └─ zig build-exe zls_gen Debug native 1 errors
src/config_gen/config_gen.zig:967:29: error: member function expected 4 argument(s), found 3
    var request = try client.request(uri, .{ .connection = .close }, .{});
                      ~~~~~~^~~~~~~~
[ZIG_PATH]/lib/std/http/Client.zig:987:5: note: function declared here
pub fn request(client: *Client, method: http.Method, uri: Uri, headers: http.Headers, options: Options) RequestError!Request {
~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    generateVersionDataFile: src/config_gen/config_gen.zig:850:26
    main: src/config_gen/config_gen.zig:1075:13
    remaining reference traces hidden; use '-freference-trace' to see all reference traces

--- a/src/config_gen/config_gen.zig
+++ b/src/config_gen/config_gen.zig
@@ -964,15 +964,16 @@ fn httpGET(allocator: std.mem.Allocator, uri: std.Uri) !Response {
     defer client.deinit();
     try client.ca_bundle.rescan(allocator);
 
-    var request = try client.request(uri, .{ .connection = .close }, .{});
+    var request = try client.request(.GET, uri, .{ .allocator = allocator }, .{});
     defer request.deinit();
 
-    try request.do();
+    try request.start();
     // try request.finish();
+    try request.do();
 
-    if (request.response.headers.status.class() != .success) {
+    if (request.response.status.class() != .success) {
         return .{
-            .other = request.response.headers.status,
+            .other = request.response.status,
         };
     }
 

